microshift_storage/persistent_storage_microshift/microshift-storage-plugin-overview.adoc

```diff
@@ -0,0 +1,10 @@
+:_content-type: ASSEMBLY
+[id="microshift-storage-plugin-overview"]
+= MicroShift storage plugin overview 
```

Comment:
Just checking if you're using "plugin" without the hyphen here purposefully, or if it should be "plug-in" like the rest of the OCP docs.

Related - the SSG did actually just change to recommending "plugin" without the hyphen, but if want to do that in OpenShift, we'll want to do a sweep across the repo to do it all at once.

Reply:
For now I will use plug-in.